README.md

```diff
@@ -51,12 +51,28 @@ graph.connect('peer1', 'peer2')
 
 > In graph theory, a directed graph (or digraph) is a graph that is a set of vertices connected by edges, where the edges have a direction associated with them.
 
-### graph = new Graph(rootElem)
+### graph = new Graph(rootElem, size)
 
 Create a new P2P graph at the root DOM element `rootElem`. In addition to an
 `Element`, a query selector string (like `'.my-cool-element'`) can also be passed
 in.
 
+Also, you can pass in an object containing the desired width and height. The possible values for both are `auto`, `default` or a number (which represents the number of pixels). Aditionally you can pass in a function that evaluates on resize.
```

Comment:
I appreciate that you tried to handle all the possible use cases that the user could have! 👍 But I think this is too many different options and we could simplify this.

Which of these options do you actually need for your use case? The only one that `webtorrent.io` needs is the ability to set a `width` and `height`. We can set it to 400 and 250 ourselves, and get rid of those magic constants in `p2p-graph`.

So, when the `size` argument is omitted let's just treat that as `auto`. And when it's specified, let's use the specified values. How does that sound?

That eliminates the `auto`, `default`, and function versions.